### PR TITLE
Implement route layouts

### LINKED ISSUE #7: Create layout component
Add a layout components that consists of a header and footer. Use children API for passing content

Goals:
- Create two layouts (main layout and secondary one for one page with no content)

Resources:
- https://www.youtube.com/watch?v=DJ53-G8EbxE
- https://shtanko-michael.medium.com/multiple-layouts-in-react-app-with-react-router-v5-2eb6a065e3c5

### PR DESCRIPTION
**Pull request by ekostandov**

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -1,15 +1,42 @@
-import Header from './Header/Header';
-import Main from './Main/Main';
-import Footer from './Footer/Footer';
+import { Switch, Route } from 'react-router-dom';
```
Comment (singvarr) [inline src/App/App.js]: move me to `components`

File: src/App/App.js [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+import ProductList from './Main/Product/ProductList';
```
Comment (singvarr) [inline src/App/App.js]: use absolute imports

File: src/App/App.js [hunk 3]
```diff
@@ -1,15 +1,42 @@
...
+          routes.CART,
+          routes.PRODUCT,
+          routes.PAYMENT,
+          routes.SHIPPING,
+        ]}
+      >
+        <MainLayout>
+          <Route exact path={routes.HOME} component={ProductList} />
+          <Route exact path={routes.PRODUCT} component={ProductPage} />
+          <Route path={routes.CART} component={CartPage} />
```
Comment (singvarr) [inline src/App/App.js]: Why these pages aren't exact?

File: src/App/App.js [hunk 4]
```diff
@@ -1,15 +1,42 @@
...
+          <Route path={routes.PAYMENT} component={PaymentPage} />
+          <Route path={routes.SHIPPING} component={ShippingPage} />
+        </MainLayout>
+      </Route>
+
+      <Route>
+        <SecondaryLayout></SecondaryLayout>
```
Comment (singvarr) [inline src/App/App.js]: Add a dummy page that uses the second layout, please

File: src/layouts/MainLayout.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+//import React from 'react';
```
Comment (singvarr) [inline src/layouts/MainLayout.js]: remove comment

**Review by singvarr — COMMENTED** [review]
In general looks good but let's add some minor improvements

File: src/App/App.js (hunk 2 above)
Comment (ekostandov) [reply src/App/App.js]: Done.

File: src/App/App.js (hunk 3 above)
Comment (ekostandov) [reply src/App/App.js]: Done.

File: src/layouts/MainLayout.js (hunk 5 above)
Comment (ekostandov) [reply src/layouts/MainLayout.js]: Done.

File: src/App/App.js (hunk 4 above)
Comment (ekostandov) [reply src/App/App.js]: Done.

File: src/App/App.js (hunk 1 above)
Comment (ekostandov) [reply src/App/App.js]: Done.

**Review by singvarr — APPROVED** [review]

**Pull request merged**